### PR TITLE
Apr 2174 removed apps in oversight

### PR DESCRIPTION
**Pull request by chrisfoster76**

### CODE REVIEW

File: src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs [hunk 1]
```diff
@@ -1,12 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Text;
-
-namespace SFA.DAS.RoatpOversight.Domain
+﻿namespace SFA.DAS.RoatpOversight.Domain
 {
-    public static class GatewayReviewStatus
+    public class GatewayReviewStatus
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs:3]: Should be a static class as it contains only contants

File: src/SFA.DAS.RoatpOversight.Web/Views/Shared/_GatewayOutcome.cshtml [hunk 2]
```diff
@@ -21,6 +21,12 @@
                     Passed
                 </strong>
             }
+            else if (Model.GatewayReviewStatus == GatewayReviewStatus.Removed)
+            {
+        <strong class="govuk-tag govuk-tag--inactive">
+            Removed
+        </strong>
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpOversight.Web/Views/Shared/_GatewayOutcome.cshtml:28]: small niggle but the indentation is out with the rest of the markup

File: src/SFA.DAS.RoatpOversight.Web/Views/Shared/_GatewayOutcome.cshtml [hunk 3]
```diff
@@ -60,7 +66,7 @@
             }
         </dd>
     </div>
-    @if (Model.GatewayReviewStatus == GatewayReviewStatus.Fail)
+    @if (Model.GatewayReviewStatus != GatewayReviewStatus.Pass)
```
Comment (martin-lush) [inline src/SFA.DAS.RoatpOversight.Web/Views/Shared/_GatewayOutcome.cshtml:69]: Is this intentional? I though the list was more than Pass/Fail/Removed ?

File: src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs (hunk 1 above)
Comment (martin-lush) [reply src/SFA.DAS.RoatpOversight.Domain/GatewayReviewStatus.cs:3]: Feels like this class is lacking some of the other statuses. For example
 
"In Progress"
"Clarification sent"

**Review by martin-lush — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.RoatpOversight.Web/Views/Shared/_GatewayOutcome.cshtml (hunk 3 above)
Comment (chrisfoster76) [reply src/SFA.DAS.RoatpOversight.Web/Views/Shared/_GatewayOutcome.cshtml:69]: Yes, the block shows if it's anything other than Pass

**Review by martin-lush — APPROVED** [review]

**Pull request merged**